src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFromFile(fromFileName);
+        String report = getReport(dataFromFile);
+        writeToFile(toFileName, report);
+    }
+
+    private static String getReport(String fromFileName) {
+        String[] words = fromFileName.split(",");
+        StringBuilder stringBuilder = new StringBuilder();
+        int resultSupply = 0;
+        int resultBuy = 0;
+        for (int i = 0; i < words.length; i++) {
+            if (words[i].equals("supply")) {
+                resultSupply += Integer.parseInt(words[i + 1]);
+            }
+            if (words[i].equals("buy")) {
+                resultBuy += Integer.parseInt(words[i + 1]);
+            }
+        }
+        int res = resultSupply - resultBuy;
+        StringBuilder builder = new StringBuilder();
+        String report = builder.append("supply").append(",").append(String.valueOf(resultSupply))
+                .append(System.lineSeparator())
+                .append("buy").append(",").append(String.valueOf(resultBuy))
+                .append(System.lineSeparator())
+                .append("result").append(",").append(String.valueOf(res)).toString();
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)